### DISCUSSION TITLE
failed to get status error

### DISCUSSION THREAD

**Discussion by m4jid-ha** [metalmatze/alertmanager-bot#183] [Help/Support] [answered]
Hi everyone

I did all the config step by step and now my bot container with docker-compose is up and alertmanager set in .yml file and for now up on http://localhost:9093 on my server . but when i want to communicate whit my bot response this error:

" failed to get status... Get http://localhost:9093/api/v1/status: dial tcp 127.0.0.1:9093: connect: connection refused"

tnx for help

**Comment by metalmatze** [comment, marked as answer]
This looks like the alertmanager-bot is unable to talk to the alertmanager. Since you have mentioned docker-compose you need to use the alertmanager container's name instead of localhost to be able to reach it.
I hope that helps.
